README.md

```diff
@@ -1,52 +1,49 @@
 
-| Component                  | Source Code                                                                   |
-| -------------------------- | ----------------------------------------------------------------------------- |
-| KServe                     | [`kserve/kserve`](https://github.com/kserve/kserve)                           |
-| Kubeflow Katib             | [`kubeflow/katib`](https://github.com/kubeflow/katib)                         |
-| Kubeflow Model Registry    | [`kubeflow/model-registry`](https://github.com/kubeflow/model-registry)       |
-| Kubeflow MPI Operator      | [`kubeflow/mpi-operator`](https://github.com/kubeflow/mpi-operator)           |
-| Kubeflow Notebooks         | [`kubeflow/notebooks`](https://github.com/kubeflow/notebooks)                 |
-| Kubeflow Pipelines         | [`kubeflow/pipelines`](https://github.com/kubeflow/pipelines)                 |
-| Kubeflow Spark Operator    | [`kubeflow/spark-operator`](https://github.com/kubeflow/spark-operator)       |
-| Kubeflow Training Operator | [`kubeflow/training-operator`](https://github.com/kubeflow/training-operator) |
+## Kubeflow Platform
 
-If you want to open issue or pull request for the
-[Kubeflow Platform](https://www.kubeflow.org/docs/started/introduction/#what-is-kubeflow-platform)
-components:
+The [Kubeflow Platform](https://www.kubeflow.org/docs/started/introduction/#what-is-kubeflow-platform) refers to the full suite of Kubeflow Components bundled together with additional integration and management tools.
 
-- Use the [`kubeflow/manifests`](https://github.com/kubeflow/manifests) GitHub repository for
-  the Kubeflow Manifests.
-- Use the [`kubeflow/dashboard`](https://github.com/kubeflow/dashboard) GitHub repository for
-  the Kubeflow Profile Controller, Central Dashboard, CRUD Web Apps, PVC Viewer, PodDefault, and
-  Access Management components.
+The following table lists the platform components and their respective source code repositories:
 
-If you have questions about Kubeflow community or Kubeflow ecosystem, please use the
-[`kubeflow/community`](https://github.com/kubeflow/community) GitHub repository.
+| Component                                                                             | Source Code                                                   |
+|---------------------------------------------------------------------------------------|---------------------------------------------------------------|
+| [Central Dashboard](https://www.kubeflow.org/docs/components/central-dash/)           | [`kubeflow/dashboard`](https://github.com/kubeflow/dashboard) |
+| [Profile Controller](https://www.kubeflow.org/docs/components/central-dash/profiles/) | [`kubeflow/dashboard`](https://github.com/kubeflow/dashboard) |
+| Kubeflow Manifests                                                                    | [`kubeflow/manifests`](https://github.com/kubeflow/manifests) |
```

Comment:
Should we link the README for Kubeflow Manifests: https://github.com/kubeflow/manifests?tab=readme-ov-file#overview-of-the-kubeflow-platform ?

Reply:
It's the same link as the repo, also, we need to be careful about linking people directly to the `master` of manifests, as they might try and install from there, rather than an actual version like `v1.9.0` tag.

Reply:
Sure, @kubeflow/wg-manifests-leads any comments there ?
I am fine to make this as a followup change when we have dedicated website section that explains Kubeflow Manifests.

Reply:
@andreyvelich I don't think we need more than we already have on the "Installing Kubeflow" page: 
- https://www.kubeflow.org/docs/started/installing-kubeflow/#kubeflow-manifests

As I keep highlighting, the manifests are intended to be used as the base of distribution, and by advanced users wanting to test Kubeflow out.

For almost everyone else, they will have a bad experience trying to use the manifests in production. We already get a lot of spam issues from people asking for help on issues that are unrelated to our Kubeflow and are more about their specific platform, I don't want to increase the amount of that.

Reply:
I have added the link above in https://github.com/kubeflow/kubeflow/pull/7642/commits/7abcd7727c4611076aa22c4d63eced167e14b619